common/src/Microsoft.Azure.IIoT.Serializers.NewtonSoft/src/Default/NewtonSoftJsonSerializer.cs

```diff
@@ -232,7 +232,7 @@ internal class JsonVariantValue : VariantValue {
             public override object ConvertTo(Type type) {
                 try {
                     return Token.ToObject(type,
-                        JsonSerializer.CreateDefault(_serializer.Settings));
+                        JsonSerializer.CreateDefault(_serializer.Settings));                   
```

Comment:
Undo this file - (or remove added whitespace)

Reply:
done